controllers/index.js

```diff
@@ -4,14 +4,16 @@ const paragraph = require("../public/javascripts/data/paragraph");
 // GET request
 // displayAllParagraphs function returns an array of all paragraphs in database
 exports.index = function(req, res, next) {
-
-  res.render('index', { ps: paragraph.displayAllParagraphs() });
+  
+  let currentUser = res.locals.userName;
```

Comment:
I think this variable would be better named as `currentUserName`, since it's not the entire user object that is stored in `locals`